local-dev/templates/rawls.conf.ctmpl

```diff
@@ -0,0 +1,460 @@
+{{- $appsDomain := "test.firecloud.org" -}}
+{{- $appsSubDomain := printf "dev.%s" $appsDomain -}}
+{{- $orgId := "400176686919" -}}
+{{- $b2cEnv := "dev" -}}
+{{- $b2cTenant := "dev" -}}
+
+{{- $rawlsOauthCredential := (secret "secret/dsde/firecloud/dev/rawls/rawls-oauth-credential.json").Data }}
+{{- $rawlsConf := (secret "secret/dsde/firecloud/dev/rawls/rawls.conf").Data -}}
+{{- $refreshTokenCredential := (secret "secret/dsde/firecloud/dev/common/refresh-token-oauth-credential.json").Data -}}
+{{- $rawlsSaKey := (secret "secret/dsde/firecloud/dev/rawls/rawls-account.json").Data -}}
+{{- $cromwellSaKey := (secret "secret/dsde/firecloud/dev/cromwell/cromwell-account.json").Data -}}
+{{- $staticPerimeterProjects := (secret "secret/dsde/firecloud/dev/rawls/service-perimeters/static-projects").Data -}}
+{{- $b2cAppId := (secret "secret/dsde/terra/azure/dev/b2c/application_id" ).Data.value -}}
+{{- $landingZonePostgres := (secret "secret/dsde/terra/azure/dev/workspacemanager/azure-postgres-credential" ).Data -}}
+
+backRawls = true
```

Comment:
for local development, this should be false by default. I think that's how we have it in firecloud-develop.

Reply:
@davidangb When I run `./run-context/local/scripts/firecloud-setup.sh` and select `rawls` (as described in the README), it produces a `rawls.conf` with `backRawls = true`. What process do you use generate the config templates? Maybe I'm missing some additional local config?

Reply:
ah, I figured it out - the docker script overrides it via system property: https://github.com/broadinstitute/rawls/pull/2383/files#diff-ae2e558e0e238c2247080ce69bad049968b7643f2411424256631f556489f45aR74

In any case, I'll adjust the template so it's less confusing